cci_nepal/pipeline/classification_model/model_save_run/model_run.py

```diff
@@ -0,0 +1,142 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.2
+#   kernelspec:
+#     display_name: cci_nepal
+#     language: python
+#     name: cci_nepal
+# ---
+
+# %% [markdown]
+# # Model run
+# <br>
+# <br>
+# The purpose of this script is to run the x2 classification models (basic and non-basic NFRI items) on new data (defaulted to the test set) and save the results to excel files. The resulting files are:
+#
+# - `basic_test_predictions.xlsx`
+# - `non_basic_test_predictions.xlsx`
+#
+# The classification models are loaded from saved pretrained models which are created in the `model_save.py` file in `pipeline/classification_model/`.
+#
+# This script relies on test datasets (one for Hill and one for Terai regions) which are subsets of the survey data collected in Mahottari and Sindhupalchok. The splitting of the data is done in the data_splitting_survey.py file in `pipeline/classification_model/`.
+
+# %%
+# Import libraries
+import pandas as pd
+from sklearn.utils import shuffle
+from sklearn.compose import ColumnTransformer
+from sklearn.preprocessing import OneHotEncoder
+from sklearn.preprocessing import RobustScaler
+from sklearn.multioutput import MultiOutputClassifier
+from sklearn.linear_model import LogisticRegression
+import pickle
+
+# Project libraries
+import cci_nepal
+from cci_nepal.getters.classification_model import get_real_data as grd
```

Comment:
As folders were renamed, this needs to be `from cci_nepal.getters.classification_model import get_data as grd`